include/fastdds/rtps/transport/TransportDescriptorInterface.h

```diff
@@ -52,13 +59,19 @@ struct TransportDescriptorInterface
      */
     virtual TransportInterface* create_transport() const = 0;
 
-     //! Returns the minimum size required for a send operation.
+    //! Returns the minimum size required for a send operation.
     virtual uint32_t min_send_buffer_size() const = 0;
 
     //! Returns the maximum size expected for received messages.
-    virtual uint32_t max_message_size() const { return maxMessageSize; }
+    virtual uint32_t max_message_size() const
+    {
+        return maxMessageSize;
+    }
 
-    virtual uint32_t max_initial_peers_range() const { return maxInitialPeersRange; }
+    virtual uint32_t max_initial_peers_range() const
```

Comment:
Documentation for this method

Reply:
Done in 4fcfb6feaa85204124ab730ef1a58d1ee5337c57